test/e2e_node/startup_probe_test.go

```diff
@@ -87,15 +116,15 @@ var _ = framework.KubeDescribe("StartupProbe [Serial] [Disruptive] [NodeAlphaFea
 
 		/*
 			Release : v1.16
-			Testname: Pod liveness probe, using local file, delayed by startup probe
-			Description: A Pod is created with liveness probe that uses ‘exec’ command to cat the non-existent /tmp/health file. Liveness probe MUST fail after startup probe expires. The Pod MUST now be killed and restarted incrementing restart count to 1.
+			Testname: Pod liveness probe delayed (short) by startup probe
+			Description: A Pod is created with failing liveness and startup probes. Liveness probe MUST fail after startup probe expires. The Pod MUST now be killed and restarted incrementing restart count to 1.
 		*/
-		framework.ConformanceIt("should be restarted with a exec \"cat /tmp/health\" because startup probe does not delay it long enough [NodeConformance]", func() {
+		framework.ConformanceIt("should be restarted by liveness probe because startup probe does not delay it long enough [NodeConformance]", func() {
```

Comment:
Shouldn't this container restart because startup probe fails?

Reply:
Yup, this test doesn't make sense...